tests/scanner/scanners/data/firewall_test_rules.yaml

```diff
@@ -0,0 +1,98 @@
+rules:
```

Comment:
Is there any reason to duplicate this file here, instead of just pointing to the other version under ../audit/data/?

Reply:
Done.